src/WordSearchToDataBase.java

```diff
@@ -0,0 +1,87 @@
+    String dateAndTimeFormat = "dd/MM/yyyy HH:mm:ss";
+    String createTable = "create table audit (PathToTheFile varchar(100), SearchedWord varchar(45), DateAndTimeOfSearch varchar(45), Result varchar(45), WordCount int, ErrorMessage varchar(100))";
+
+    /*
+    Calling connectionToDatabase method to load the driver and establish the connection.
+    Checking if the Table audit is already exists, if exists we are inserting the values into the table.
+    And closing the connection.
+    If table is not present then calling createNewTableAndInsertValues method so to creat a new table and insert the values.
+     */
+    public void dataBaseStorage(String filePath, String wordSearched, String theResult, int wordCount, String errorMessage) throws SQLException {
+        Connection connectionToDataBase = null;
+        Statement statementQueries;
+        DateTimeFormatter dateTimeFormat = DateTimeFormatter.ofPattern(this.dateAndTimeFormat);
+        LocalDateTime now = LocalDateTime.now();
+        String presentDateAndTime = dateTimeFormat.format(now);
+        try {
+            connectionToDataBase = connectionToDatabase();
+            statementQueries = connectionToDataBase.createStatement();
+            DatabaseMetaData checkIfTableIsThere = connectionToDataBase.getMetaData();
+            ResultSet tables = checkIfTableIsThere.getTables(null, null, Constants.AUDIT, null);
+            if (tables.next()) {
+                statementQueries.execute(insertValuesToTable(filePath, wordSearched, presentDateAndTime, theResult, wordCount, errorMessage));
+            } else {
+                createNewTableAndInsertValues(filePath, wordSearched, presentDateAndTime, theResult, wordCount, errorMessage);
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        } finally {
+            Objects.requireNonNull(connectionToDataBase).close();
+        }
+    }
+
+    /*
+    Calling connectionToDatabase method to load the driver and establish the connection.
+    Creating a new table and insert the values into new table.
+     */
+    private void createNewTableAndInsertValues(String filePath, String wordSearched, String presentDateAndTime, String theResult, int wordCount, String errorMessage) throws SQLException {
+        Connection connectionToDataBase = connectionToDatabase();
+        try {
+            Statement statementQueries = connectionToDataBase.createStatement();
+            statementQueries.execute(this.createTable);
+            statementQueries.execute(insertValuesToTable(filePath, wordSearched, presentDateAndTime, theResult, wordCount, errorMessage));
+        } catch (Exception e) {
+            e.printStackTrace();
+        } finally {
+            Objects.requireNonNull(connectionToDataBase).close();
+        }
+    }
+
+    /*
+    Loading the Driver for MySQL and Establishing the connection.
+     */
+    private Connection connectionToDatabase() throws SQLException {
```

Comment:
Method name not proper. Java method names should resemble commands

```suggestion
    private Connection connectToDatabase() throws SQLException {
```

Reply:
Updated.